gettsim/transfers/arbeitsl_geld_2/kost_unterk.py

```diff
@@ -21,6 +22,48 @@ def arbeitsl_geld_2_kost_unterk_m_hh(
     )
 
 
+def arbeitsl_geld_2_kost_unterk_m_hh_ab_2023(
+    bruttokaltmiete_m_hh: float,
+    heizkosten_m_hh: float,
+    bürgerg_bezug_vorj: float,
+    _arbeitsl_geld_2_berechtigte_wohnfläche_hh: int,
+    _arbeitsl_geld_2_warmmiete_pro_qm_hh: float,
+) -> float:
+    """Calculate costs of living eligible to claim since 2023.
+    During the first two years, the waiting period, the appropriateness
+    of the living costs is not tested but considered fully in Bürgergeld.
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+
+    Parameters
+    ----------
+    bruttokaltmiete_m_hh
+        See basic input variable :ref:`bruttokaltmiete_m_hh <bruttokaltmiete_m_hh>`.
+    heizkosten_m_hh
+        See basic input variable :ref:`heizkosten_m_hh <heizkosten_m_hh>`.
+    bürgerg_bezug_vorj
+        See basic input variable :ref:`bürgerg_bezug_vorj <bürgerg_bezug_vorj>`.
+    _arbeitsl_geld_2_berechtigte_wohnfläche_hh
+        See :func:`_arbeitsl_geld_2_berechtigte_wohnfläche_hh`.
+    _arbeitsl_geld_2_warmmiete_pro_qm_hh
+        See :func:`_arbeitsl_geld_2_warmmiete_pro_qm_hh`.
+
+    Returns
+    -------
+    float with total monthly cost of rent.
+    """
+    if bürgerg_bezug_vorj == 0:
```

Comment:
Is this also the Karenzzeit or a different concept? (I thought Karenzzeit is only for 1 year). If it is something different, we might need another input variable. Or at least comment about it. Let's discuss it in this case.

Reply:
As I understood this, this is also just during the Karenzzeit ([Chapter 7](https://www.betanet.de/buergergeld.html)). It is not clear to me why it should matter that it is only 1 year?